.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: continuous-integration
+on:
+  pull_request:
+    branches: [main]
+jobs:
+  conventional-commits:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
@lucky-edu: this is very expensive GH job/step definition - much better is merge all step to one GH workflow (pricing is round to minutes per every GH job)

Reply:
@mmichna pls don't write these type of commnents into PRs where all resolved conversations are required to pass checks